editor/components/post-text-editor/index.js

```diff
@@ -18,6 +18,21 @@ import { withInstanceId } from '@wordpress/components';
  */
 import './style.scss';
 
+/**
+ * Returns the PostTextEditor state given a set of props.
+ *
+ * @param {Object} props Component props.
+ *
+ * @return {Object} State object.
+ */
+function computeDerivedState( props ) {
+	return {
+		persistedValue: props.value,
```

Comment:
This one is tricky, but I see now why it exists.

Reply:
>This one is tricky, but I see now why it exists.

It's unclear to me. Should we have left an inline comment?